ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+open class ContactView: UIView {
+    private struct Constants {
+        static let contactWidth: CGFloat = 70.0
+        static let avatarViewWidth: CGFloat = 70.0
+        static let avatarViewHeight: CGFloat = 70.0
+        static let firstNameLabelMinimumHeight: CGFloat = 12.0
+        static let spacingBetweenAvatarAndFirstLabel: CGFloat = 13.0
+    }
+
+    private var avatarView: AvatarView
```

Comment:
>var [](start = 12, length = 3)

can this be let? we always assign it on init

Reply:
Yes, that makes more sense